Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ruby:3.1-bullseye
 
+RUN apt-get update; apt-get install nodejs npm -y
```

Comment:
I think the default `nodejs` version that ships with Debian is [version 12](https://packages.debian.org/bullseye/nodejs), which is out of support. I typically add it to a ruby image with the following, which is derived from the [node debian installation docs](https://github.com/nodesource/distributions#debinstall):

```dockerfile
ENV NODE_MAJOR_VERSION 18

RUN curl --silent --show-error --location --retry 5 --retry-connrefuse --retry-delay 4 https://deb.nodesource.com/setup_${NODE_MAJOR_VERSION}.x | bash - \
  && apt-get update \
  && apt-get install -y --quiet --no-install-recommends \
  nodejs
```

Reply:
switched to that